src/openforms/urls.py

```diff
@@ -34,6 +35,11 @@
         admin.site.admin_view(EmailTestAdminView.as_view()),
         name="admin_email_test",
     ),
+    path(
+        "admin/logging/list/",
+        admin.site.admin_view(TimelineLogView.as_view()),
+        name="admin_logging",
+    ),
```

Comment:
I'd prefer adding this route to the `get_urls` of the `ModelAdmin` for `TimelineLogProxy` model.